src/vaults/mainnet.json

```diff
@@ -155,6 +155,11 @@
       "slug": "kodiak-bread-ohm",
       "stakeTokenAddress": "0xd9a747880393f7c33cEf1aea36909b36d421F7E5"
     },
+    {
+      "beraRewardsVault": "0x359424085d84c490a7b276e4bc72cef90a04068b",
+      "slug": "kodiak-bread-ohm",
+      "stakeTokenAddress": "0x54c603173CA92A42B81f8838d705fd3E0f98D5D4"
```

Comment:
the berarewards vault and the stake token address point to a kodi-bread-lbgt pool